bigflow/bigquery/__init__.py

```diff
@@ -1,13 +1,17 @@
-from .interactive import interactive_component as component
-from .interactive import add_label_component, sensor_component, INLINE_COMPONENT_DATASET_ALIAS
-from .dataset_configuration import DatasetConfig
-from .job import Job
 
-__all__ = [
-    'component',
-    'add_label_component',
-    'sensor_component',
-    'INLINE_COMPONENT_DATASET_ALIAS',
-    'DatasetConfig',
-    'Job'
-]
+try:
+    from .interactive import interactive_component as component
+    from .interactive import add_label_component, sensor_component, INLINE_COMPONENT_DATASET_ALIAS
+    from .dataset_configuration import DatasetConfig
+    from .job import Job
+    __all__ = [
+        'component',
+        'add_label_component',
+        'sensor_component',
+        'INLINE_COMPONENT_DATASET_ALIAS',
+        'DatasetConfig',
+        'Job'
+    ]
+except ImportError:
+    pass
```

Comment:
In this case `__all__` should be empty I think.

Reply:
and it is :)

Reply:
Done